yarn.lock

```diff
@@ -8971,6 +8963,21 @@ ethers@5.0.32:
     "@ethersproject/web" "5.0.14"
     "@ethersproject/wordlists" "5.0.10"
 
+ethers@^4.0.0-beta.1, ethers@^4.0.32:
```

Comment:
Why do we have ethers 4 again ?

Reply:
web3 and truffle depend on it apparently
This will be fixed by another PR where I remove web3 😁